### ISSUE TITLE
header linked to engine load rather than thresher?

### ISSUE THREAD

**Issue by miracleman51** [yumi-modding/FS22_CombineXP#28]
Hi,  loving the mod however one key things stands out for me which I think is incorrect. The combine revs up and is loaded when you switch on the reel, rather than the thresher... this is the wrong way around as IRL it would the the threshing system that would demand the majority of engine power and speed.  The reel/cutter bar etc would likely onluy require 10-30 HP IMO.

I run with VCA, Engine Load HUD display mod, REA22 Implements mods (amongst other things) and I find that rarely Im engine power limited - normally its the 'combine experience KW or %load ' that is the limiting factor... is this intentional.  I can see that being the case in some situations but I would have thought engine load would still be a factor in heavy crops/going uphill/with the chaff chopper on etc.

Thanks

Matt

**Comment by yumi-modding** [comment]
Hello @miracleman51 
Regarding the combine power load i did not really made something, it's default power set by Giants in game but I agree it's not realistic.
I don't for how long you know and play Farming Simulator, but on FS17 (and before) there was a great mod called moreRealistic (from which I reuse some code) that was also managing correctly power requirements for combines, cutter, chopper... The thing was that all this data needed to be defined in xml files and so required work to set everything fine (but of course infinite customization).
For now there is nothing about this in my mod and so power requirement is not managed as it should. I agree it would enhance the experience and it could be added, but maybe just with proportional value for all combines to have something easier to set, or at least default value + capability to overwrite them.
Thanks for the feedback by the way.

**Comment by miracleman51** [comment]
Hi,

Thanks for the feedback.   I think I understand what your saying.

At the moment whne you 'press B' to activate the thresher on the combine itself - not much happens - when you do the same for the header it 'revs up'.  IMO it should be the other way around.  e.g. its 'activating the thresher' that should make the combine rev up.  as the header takes very little power in real life.  would it be possible to swap this?

**Comment by kilbouri** [comment]
I agree, it is correct that the thresher is the primary load in a combine. However, you wouldn't want to start the header while at full RPMs as it would create a shock load and potentially damage the header.

That being said, I've only real-life experience with older models of combine so it is entirely possible that newer machines can be started at high RPM without any issue. Feel free to correct me if I am wrong about newer machines, I'd love to know :)

**Comment by yumi-modding** [comment]
Hello @kilbouri 
I think you're right at least for older combines, i also observe that driver set full rev after engaging the header.
But for more recent combine like at least Lexion 540 I know that it perfectly manages the fact to start header at half rev and full rev thanks to a system that engage the header progressively.
